entrypoint.sh

```diff
@@ -15,12 +16,7 @@ sh -c "git config --global core.askPass /cred-helper.sh"
 sh -c "git config --global credential.helper cache"
 sh -c "git remote add mirror $*"
 sh -c "echo pushing to $branch branch at $(git remote get-url --push mirror)"
-sh -c "git push mirror $branch"
-
```

Comment:
Why are you removing this block? Isn't this what you added in your last PR?

Reply:
This is now covered by "set -euo pipefail"